common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -0,0 +1,178 @@
+#### Local Types ####
```

Comment:
Drakkari don't know sea-faring.
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>